### ISSUE TITLE
Improve accessibility and color contrast

### ISSUE THREAD

**Issue by Ig-Rebollo** [kobotoolbox/kpi#2776]
@tinok @jnm @dorey 

Currently, the platform interface uses a variety of greys for text and background that fail to meet standard web accessibility requirements. When testing text colour over several background-color from the KPI UI on a contrast checker such as [contraschecker.com](https://contrastchecker.com/), we see most instances don't pass any of the 6 criterion. See for example the color of the Deployed/Draft/Archived menu against its light grey background:
![image](https://user-images.githubusercontent.com/22472326/91757581-91194780-eb9c-11ea-83ae-adc59e3c75b0.png)

Though this is maybe not something we should obsess about, I would at least aim to fulfill the minimum requirements (AA criterion for fonts below and above 18pt, and the color brightness pass). If we can also make sure the combination works for those with color blindness(sixth criterion), even better.

There are several ways to do this, but my recommendation would be to start by **removing all grey backgrounds**, at least for the most part, and increase the darkness of the text. Related to this issue, Leszek and I have been working on reducing the number of colors in place in the platform -particularly the sheer number of shades of grey- and have a limited number of colors in use with a clear logic. So bearing this in mind, removing most grey backgrounds would help us three ways:
1. Improve accessibility and color contrast issues
2. Reduce the number of greys in place and set a reduced palette with a more solid logic
3. Update the look of the platform to be more current. Most similar platforms have moved away from grey backgrounds for the reasons mentioned above.

@magicznyleszek is looking into this and how much work it would imply. For the time being it is just in the backlog, though depending on the complexity of it and our timelines we could consider addressing this sooner than later. Below I attach an example of how the UI could look like without changing the structure and layout, only the necessary elements to increase color contrast (with just a few other minor updates you can disregard for now).
Current look:
![image](https://user-images.githubusercontent.com/22472326/91758585-400a5300-eb9e-11ea-87a7-db618fc93117.png)
With enhanced color contrast and related improvements:
![image](https://user-images.githubusercontent.com/22472326/91758519-279a3880-eb9e-11ea-9396-e0a772860616.png)

**Comment by tinok** [comment]
Let's definitely do all the accessibility stuff, color cleanup, and contrasts. 

In the proposed screen I noticed some other changes, it would be good to break these out into distinct issues so we can discuss them there (as not related to accessibility). Some quick feedback:
* The Q1 Q2 etc in the table header probably wouldn't work. There is no question number system that's visible at any point in the form creation process, and users may have a hard quarter to question number in the question label.
* Replacing the X button with a back to projects button makes sense, as discussed before the X button is quite confusing to some users. What is the reason for moving it to the left corner since users are used to the functionality being in the top right corner?

**Comment by jnm** [comment]
Hey all, once the colors are finalized, please reassign @magicznyleszek to make the changes :+1:

**Comment by magicznyleszek** [comment]
Closing this as all the big changes were implemented. Let's create a separate issue when we discover something worth doing on its own :)
